package.json

```diff
@@ -35,8 +35,8 @@
     "clean": "rm -rf assets/stylesheets build dist node_modules"
   },
   "engines": {
-    "node": "^18",
-    "npm": "^9"
+    "node": "^20",
+    "npm": "^10"
```

Comment:
i’d recommend removing the `engine.npm` field in package.json cos i don’t think it’s useful and we don’t do anything special with npm

Reply:
Done here: https://github.com/ministryofjustice/hmpps-incentives-ui/pull/627/commits/90c8f3cf2c4fc567d7f1516c109dfbddd3c07292

Reply:
i was suggesting getting rid of just `npm`, so that you'd still get a warning is using an unexpected `node` version. but frankly it's only a warning and doesn't _stop_ you using a version of node that we don't test on 🤷

Reply:
This should be better then 🤞 : https://github.com/ministryofjustice/hmpps-incentives-ui/pull/627/commits/1add893923baf6238d6d5d3c5c2a918c7317b460